NeuroML2/GenerateNeuroMLlite.py

```diff
@@ -912,8 +931,24 @@ def internal_connections(pops, W, pre_pop, post_pop):
             #Load the array with the ordered rank
             import pandas as pd
 
-
-            areas = ['V1', 'V4', 'MT', 'TEO']
+            if '-3rois' in sys.argv:
+                areas = ['V1', 'V4', 'MT']
+                # Create the FB and FF connectiviy between the areas.
+                # Define connections between regions
+                # NOTE: This analysis assumes that the matrix has been organised by the areas ranking
+                conn = np.array([[0, 1, 0],
+                                 [1, 0, 1],
+                                 [0, 1, 0]])
```

Comment:
@JessyD is this correct? The previous version of this matrix didn't need the 1s in both the upper half & lower half...

Reply:
Yes, that is correct.  I split the FF and FB connections as it is more intuitive to have them separately. Now we can have only FB connections to one region (e.g. there is only a FB between TEO and MT in the [4rois example](https://github.com/OpenSourceBrain/MejiasEtAl2016/commit/c9558b2444a7e13d497f82221877812b6940102f#comments))